packages/flutter/lib/src/rendering/tweens.dart

```diff
@@ -23,3 +28,29 @@ class FractionalOffsetTween extends Tween<FractionalOffset> {
   @override
   FractionalOffset lerp(double t) => FractionalOffset.lerp(begin, end, t);
 }
+
+/// An interpolation between two [FractionalOffsetGeometry].
+///
+/// This class specializes the interpolation of [Tween<FractionalOffsetGeometry>]
+/// to be appropriate for rectangles.
+///
+/// See [Tween] for a discussion on how to use interpolation objects.
+///
+/// See also:
+///
+///  * [FractionalOffsetTween], which interpolates between two
+///    [FractionalOffset] objects.
+class FractionalOffsetGeometryTween extends Tween<FractionalOffsetGeometry> {
+  /// Creates a fractional offset geometry tween.
+  ///
+  /// The [begin] and [end] properties may be null; the null value
+  /// is treated as meaning the top left corner.
```

Comment:
I don't believe this is correct. I think it is interpreted as the top center, or maybe center center.

Reply:
Looks like the center.